.github/workflows/pull_request.yml

```diff
@@ -21,6 +21,43 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          path: "."
+          pattern: "*.sh"
+
+      - uses: cachix/install-nix-action@v22
+      - uses: rrbutani/use-nix-shell-action@v1.1.0
+        with:
+          file: shell.nix
+
+      - name: Set up Go 1.19
+        uses: actions/setup-go@v5
```

Comment:
We should have same version of this workflow throughout?
Maybe we could bump the other ones to v5 as well..

Reply:
Done